capabilities.json

```diff
@@ -37,6 +37,10 @@
             "pattern" : "^/halp",
             "url" : "http://imgur-hipchat.herokuapp.com",
             "event" : "room_message" } ],
+            {
+            "pattern" : "^/roll .+",
```

Comment:
Do you want hipchat to send dankBot messages that start with `/roll` or only messages that contain something afterwards like `/roll stuff here` ?

Because if you look at the pattern for `/halp`, you'll see it looks a little different.